examples/deadline/Local-Zone/ts/lib/compute-tier.ts

```diff
@@ -0,0 +1,114 @@
+ */
+
+import {
+  BastionHostLinux,
+  IMachineImage,
+  InstanceClass,
+  InstanceSize,
+  InstanceType,
+  IVpc,
+  SubnetSelection,
+  SubnetType,
+} from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import {
+  HealthMonitor,
+  IHealthMonitor,
+  SessionManagerHelper,
+} from 'aws-rfdk';
+import {
+  IRenderQueue,
+  IWorkerFleet,
+  WorkerInstanceFleet,
+} from 'aws-rfdk/deadline';
+
+/**
+ * Properties for {@link ComputeTier}.
+ */
+export interface ComputeTierProps extends cdk.StackProps {
+  /**
+   * The VPC to deploy resources into.
+   */
+  readonly vpc: IVpc;
+
+  /**
+   * The availability zones the worker instances will be deployed to. This can include your local
+   * zones, but they must belong to the same region as the standard zones used in other stacks in
+   * this application.
+   */
+  readonly availabilityZones: string[],
+
+  /**
+   * The {@link IRenderQueue} that Deadline Workers connect to.
+   */
+  readonly renderQueue: IRenderQueue;
+
+  /**
+   * The {@link IMachineImage} to use for Workers (needs Deadline Client installed).
+   */
+  readonly workerMachineImage: IMachineImage;
+
+  /**
+   * The name of the EC2 keypair to associate with Worker nodes.
+   */
+  readonly keyPairName?: string;
+
+  /**
+   * The bastion host to allow connection to Worker nodes.
+   */
+  readonly bastion?: BastionHostLinux;
+}
+
+/**
+ * The computer tier consists of the worker fleets. We'll be deploying the workers into the
+ * local zone we're using.
+ */
+export class ComputeTier extends cdk.Stack {
+  /**
+   * The {@link IWorkerFleet}.
+   */
+  public readonly workerFleet: IWorkerFleet;
+
+  /**
+   * The {@link IHealthMonitor} used to maintain the worker fleet.
+   */
+  public readonly healthMonitor: IHealthMonitor;
+
+  /**
+   * Initializes a new instance of {@link ComputeTier}.
+   */
+  constructor(scope: cdk.Construct, id: string, props: ComputeTierProps) {
+    super(scope, id, props);
+
+    // We can put the health monitor and worker fleet in all of the local zones we're using
+    const subnets: SubnetSelection = {
+      availabilityZones: props.availabilityZones,
+      subnetType: SubnetType.PRIVATE,
+      onePerAz: true,
+    };
+
+    this.healthMonitor = new HealthMonitor(this, 'HealthMonitor', {
+      vpc: props.vpc,
+      vpcSubnets: subnets,
+      deletionProtection: false,
+    });
+
+    this.workerFleet = new WorkerInstanceFleet(this, 'WorkerFleet', {
+      vpc: props.vpc,
+      renderQueue: props.renderQueue,
+      workerMachineImage: props.workerMachineImage,
+      healthMonitor: this.healthMonitor,
+      keyName: "OpenSshDevBox",
+      // Not all instance types will be available in local zones. For a list of the instance types
+      // available in each local zone, you can refer to:
+      // https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/#AWS_Services
+      instanceType: InstanceType.of(InstanceClass.T3, InstanceSize.LARGE),
```

Comment:
nit: We're using the `InstanceType.of(InstanceClass.STANDARD5, InstanceSize.LARGE)` in the python sample but a different one here. Should we change one of them to be the same as the other for consistency?

Reply:
I had it like this because for some reason the TS and Python implementations of the `InstanceClass` enum have different entries. I did a little more digging and it seems that `T3` is equivalent to `BURSTABLE3` which is in both implementations, so I'll change them both to use that.

Reply:
Oh weird, I didn't know they had didn't enum values. `BURSTABLE3` sounds good to me.